### PR TITLE
Sui mono/autorelease

### PR DESCRIPTION
**Pull request by jcger**
## Description
Automatic releases for CI

### CODE REVIEW

File: packages/sui-mono/bin/sui-mono-release.js [hunk 1]
```diff
@@ -142,9 +147,61 @@ const checkIsMasterBranchActive = async ({status, cwd}) => {
   }
 }
 
+const execute = async (cmd, full) => {
```
Comment (carlosvillu) [inline packages/sui-mono/bin/sui-mono-release.js:150]: Where are you using this function?

Comment (kikoruiz) [reply packages/sui-mono/bin/sui-mono-release.js:150]: https://github.com/SUI-Components/sui/pull/907/files#diff-07d9dd6c75c76a2e1d0a25dc5fd3e0cbR170

Comment (kikoruiz) [reply packages/sui-mono/bin/sui-mono-release.js:150]: L 170

Comment (jcger) [reply packages/sui-mono/bin/sui-mono-release.js:150]: in line 170

File: packages/sui-mono/bin/sui-mono-release.js [hunk 2]
```diff
@@ -2,16 +2,21 @@
 require('util.promisify/shim')()
 const program = require('commander')
 const path = require('path')
+const {shell} = require('@tunnckocore/execa')
 const config = require('../src/config')
 const checker = require('../src/check')
 const {serialSpawn, showError} = require('@s-ui/helpers/cli')
 const {getPackageJson} = require('@s-ui/helpers/packages')
 const {exec: execNative} = require('child_process')
+const GitUrlParse = require('git-url-parse')
```
Comment (midudev) [inline packages/sui-mono/bin/sui-mono-release.js:11]: Shall we use camelCase for this method?

File: packages/sui-mono/bin/sui-mono-release.js [hunk 3]
```diff
@@ -142,9 +147,61 @@ const checkIsMasterBranchActive = async ({status, cwd}) => {
...
+      githubToken: program.githubToken,
+      githubUser: program.githubUser
+    })
+      ? await automaticRelease({
+          githubEmail: program.githubEmail,
+          githubToken: program.githubToken,
+          githubUser: program.githubUser,
+          cwd: process.cwd()
+        })
+      : await checkIsMasterBranchActive({status, cwd: process.cwd()})
```
Comment (midudev) [inline packages/sui-mono/bin/sui-mono-release.js:202]: Bit shorter and easier to read.

```suggestion
   const {githubEmail, githubToken, githubUser} = program
    isAutomaticRelease({
      githubEmail,
      githubToken,
      githubUser
    })
      ? await automaticRelease({
          githubEmail,
          githubToken,
          githubUser,
          cwd: process.cwd()
        })
      : await checkIsMasterBranchActive({status, cwd: process.cwd()})
```

**Pull request merged**